tests/load/test_state_integrity.py

```
@@ -132,15 +132,43 @@ def main():
-    )
+    # --- Verify initial state (retry in case state isn't indexed yet) ---
+    initial_count = None
+    max_state_retries = 20
+    state_retry_delay = 5
+    for attempt in range(max_state_retries):
+        try:
+            initial_count = client.read_contract(
+                address=contract_address, function_name="get_count"
+            )
+            break
+        except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)